birdhouse/env.local.example

```diff
@@ -334,7 +334,6 @@ export GEOSERVER_ADMIN_PASSWORD=geoserverpass
 #
 ### public-read paths
 #
-## /data/jupyterhub_user_data/public-share/
```

Comment:
Instead of removing, update with the variable? 
The example value helps understand the purpose of that variable.